packages/message-kit/src/lib/handlerContext.ts

```
@@ -36,8 +36,11 @@ export default class HandlerContext {
   v2client!: ClientV2;
   skills?: SkillGroup[];
   members?: AbstractedMember[];
+  admins?: string[];
+  superAdmins?: string[];
   sender?: any;
   getMessageById!: (id: string) => DecodedMessage | null;
+  executeSkill!: (text: string) => Promise<void | SkillResponse>;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)